tripper/mappings/mappings.py

```diff
@@ -122,7 +122,7 @@ def __repr__(self):
             args.append(f", property_iri={self.property_iri}")
         if self.cost:
             args.append(f", cost={self.cost}")
-        return f"Value({self._value}{''.join(args)})"
+        return f"Value({self._value!r}{''.join(args)})"
```

Comment:
Is this change related to the handling of literals?

Reply:
Not directly. The change was done to improve the representation of Value objects, needed while developing/debugging this PR.